pkg/release/release.go

```diff
@@ -11,10 +11,11 @@ import (
 
 	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 
-	"github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1"
+	apiV1 "github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1"
```

Comment:
I just think apiV1 is a lot clearer than `v1` given we have a `v1client` and `helmV3` floating around